src/raft/raft.rs

```diff
@@ -1160,6 +1153,16 @@ impl<T: Storage> Raft<T> {
                     self.send(to_send);
                 }
             }
+            MessageType::MsgTransferLeader => {
+                if self.leader_id == INVALID_ID {
+                    info!("{} no leader at term {}; dropping leader transfer msg",
```

Comment:
https://github.com/coreos/etcd/pull/6155/files#diff-a8629a0eb6da1c6c3805111f1f1af85eR874
Also we should print the self.id